pkg/controller/build/build_controller.go

```diff
@@ -240,6 +242,13 @@ func (r *ReconcileBuild) Reconcile(request reconcile.Request) (reconcile.Result,
 	b.Status.Registered = corev1.ConditionFalse
 	b.Status.Reason = succeedStatus
 
+	// Validate if remote repository exists
+	if output, err:= r.validateSourceUrl(ctx, b, b.Spec.Source.URL); err !=nil {
```

Comment:
I am curios about the missing space between `err` and `:=`. Any chance the Go formatter did not run over that file. In order to avoid side changes in an upcoming change to this file, I would double check running `gofmt -s -w ...` on this file.

Reply:
We should add a make verify step that runs gofmt on the files.

Reply:
took a note here, resolving